biogeochem/EDPhysiologyMod.F90

```diff
@@ -1866,22 +1879,30 @@ subroutine SeedIn( currentSite, bc_in )
                 end select
 
                 ! Seed input from external sources (user param seed rain, or dispersal model)
-                seed_in_external =  seed_stoich*EDPftvarcon_inst%seed_suppl(pft)*years_per_day
+                ! Include both prescribed seed_suppl and seed_in dispersed from neighbouring gridcells
+                seed_in_external = bc_in%seed_in(pft)/area + seed_stoich*EDPftvarcon_inst%seed_suppl(pft)*years_per_day ![kg/m2/day]
```

Comment:
@YanlanLiu @rgknox since this is inside the element loop, should we be making sure to multiply the seed in by the `seed_stoich` value?  
```suggestion
                seed_in_external = seed_stoich*(bc_in%seed_in(pft)/area + EDPftvarcon_inst%seed_suppl(pft)*years_per_day) ![kg/m2/day]
```

Reply:
Talked about this with @ckoven and he agreed this fix is necessary.